### ISSUE TITLE
[Feature Request] Allow adding evidence files by pasting an image from the clipboard

### ISSUE THREAD

**Issue by Res260** [SecurityRiskAdvisors/VECTR#194]
When adding screenshots as evidence files, it's very cumbersome to have to save the image to disk, then navigate to find it, then upload it.

A nice solution to this is to allow pasting images to upload them to vectr.

Thoughts?

**Comment by Res260** [comment]
ps: I wish vectr was open source so I could do a PR :)

**Comment by SRAPSpencer** [comment]
Good idea for a feature request, we're deep in UI work at the moment I'll pass your feedback along to the team.
